File: v2/snippets/testResources/python-package/TestClass.py
# <Test_Resource_All>
class TestClass:
    static_value = 3

    def __init__(self, public_value, private_value):
        self.public_value = public_value
        self.private_value = private_value

    @staticmethod
    def multiply_by_two(value):
        return value * 2

    def multiply_two_numbers(self, first, second):
        return first * second

# </Test_Resource_All>

File: v2/snippets/testResources/netframework-dll/TestClass.cs
//<Test_Resource_All>
namespace TestClass {
	public class TestClass {
		public int PublicValue { get; set; }
		private int PrivateValue;
		public static int StaticValue = 3;

		public TestClass() {
			PublicValue = 1;
			PrivateValue = 2;
		}

		public TestClass(int publicValue, int privateValue) {
			PublicValue = publicValue;
			PrivateValue = privateValue;
		}

		public static int MultiplyByTwo(int a) {
			return 2 * a;
		}

		public int MultiplyTwoNumbers(int first, int second) {
			return first * second;
		}
	}
}
//</Test_Resource_All>